src/components/ActionsMenu/ActionsMenu.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import styles from './ActionsMenu.st.css';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+import { ActionsMenuItem } from './Item/ActionsMenuItem';
+import { ActionsMenuDivider } from './Divider/Divider';
+
+export enum Alignment {
+  left = 'left',
+  center = 'center',
+  right = 'right',
+}
+
+export interface ActionsMenuProps {
+  alignment?: Alignment;
+  onBlur?(): void;
```

Comment:
Is it unused because it is a WIP?

Reply:
I actually told @nataliia-khitsan to add the onblur callback.
I thought it might be needed for a11y, but i'm not sure anymore.
We can remove it for now.